akka-stream/src/main/scala/akka/stream/scaladsl/OperationAttributes.scala

```diff
@@ -57,6 +63,7 @@ object OperationAttributes {
   final case class InputBuffer(initial: Int, max: Int) extends Attribute
   final case class Dispatcher(dispatcher: String) extends Attribute
   final case class SupervisionStrategy(decider: Supervision.Decider) extends Attribute
+  final case class MaterializationName(n: String) extends Attribute
```

Comment:
If we go with it probably no. I'll think of another way to get this data to my Stage though - @drewhk points out it feels quite hacky.